Helpers/NinHelper.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace HalloVerden\ValidatorConstraintsBundle\Helpers;
+
+final class NinHelper {
```

Comment:
Seeing this in the context of this package, I'm thinking we should rename to `NorwegianNinHelper` (and update other classes and properties to match.